### ISSUE TITLE
MyManga.io block the app (again)

### ISSUE THREAD

**Issue by Zykino** [raulhaag/MiMangaNu#469]
Look the same as #246: "The website you are watching in is stealing us the images."

Can I help for this?

**Comment by raulhaag** [comment]
already done for next version

**Comment by Zykino** [comment]
1) Is JapScan fixed too? (about to create an other issue).
It list the newly uploaded manga, but it don't download anything from it.

2) Also is there a way to add the annotation of the website?
a. As far as I can tell JapScan upload an english version (with a "(VUS)" tag)
b. I would love to have a notification when the tag disappear/change so I know they finally uploaded the translated version

Do I need to make an issue for 1 and or 2? (maybe you can point me to where to look at : I know web development and would like to learn more of android)

**Comment by tempura-san** [comment]
JapScan is working in the development master.

Easiest approach would be to compile and test the master version. I don't know if you have already used Android Studio, it is quite simple to use.

Unfortunately it is not possible to have the F-Droid and the self-compiled version on the device at the same time. So you need to update the app yourself (until you move back to F-Droid).

**Comment by Zykino** [comment]
Thanks, I tried renaming the app but that won't work on the long run.
I think we can move the discussion to #477.
Just a last question: If I delete the F-Droid version to install a custom build, do I loose the DB? Are the downloaded images removed?

**Comment by tempura-san** [comment]
AFAIK no - as this information is stored on the (internal) SD card. Uninstalling an app should affect only things stored in `/data/data` - where the preferences are stored, so these **will** be lost.

But I have switched over only once - so YMMV. Nonetheless it is simple to keep your fork updated via Android studio (_VCS - Git - Rebase my GitHub fork_). So it is not too much of a hassle to keep your development version up-to-date.
